ninchatsdk/src/main/java/com/ninchat/sdk/managers/NinchatConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package com.ninchat.sdk.managers;
```

Comment:
The class is still in the managers package.

Reply:
Thanks for noting. Just pushed the fixes.